### PR TITLE
Fixes for Simple UI

### PR DESCRIPTION
**Pull request by lalitpatel**
Fixes for some minor chinks in the UI.

### CODE REVIEW

**Comment by HSZemi** [comment]
Notice something? :smile: 
grafik

**Comment by lalitpatel** [comment]
Ah! The random icon size needs to be adjusted.

**Comment by HSZemi** [comment]
> Ah! The random icon size needs to be adjusted.

Yes, I think that would make sense :+1:

**Comment by lalitpatel** [comment]
I am on the road without access to my laptop and might not be able to work on this before HC4 ends.

**Comment by lalitpatel** [comment]
> Ah! The random icon size needs to be adjusted.

This is done!

File: src/components/draft/Draft.tsx [hunk 1]
```diff
@@ -204,7 +204,7 @@ class Draft extends React.Component<IProps, IState> {
 
                     {!this.state.simplifiedUI && <DraftIdInfo/>}
 
-                    {!this.state.simplifiedUI && <CivGrid civilisations={this.props.preset.civilisations}/>}
+                    <CivGrid civilisations={this.props.preset.civilisations}/>
```
Comment (HSZemi) [inline src/components/draft/Draft.tsx:207]: Why the change of mind?

Comment (lalitpatel) [reply src/components/draft/Draft.tsx:207]: We will have to either remove the Simple UI toggle for Host and Guest Captains OR show the CivGrid in Simple UI. I chose the later thinking some Captains may want to stream the draft on their stream with Simple UI. May be I am over thinking this. 🙄

Comment (lalitpatel) [reply src/components/draft/Draft.tsx:207]: There is also third option (which I did not think of earlier) – only show CivGrid on Simple UI for the Captains.

Comment (HSZemi) [reply src/components/draft/Draft.tsx:207]: > May be I am over thinking this. :roll_eyes: 

Feels like it :smile: 
Let's simply keep the CivGrid in the simple UI then, streamers can very easily crop that if they don't want to show it
